src/test/ui/issue-13483.rs

```diff
@@ -14,9 +14,3 @@ fn main() {
     } else {
     };
 }
-
-fn foo() {
-    if true {
-    } else if { //ERROR: MISSING CONDITIONAL
```

Comment:
By then we wouldn't catch, for example: 

```rust
fn foo() {
    if true {
    } else if {
    }
    bar();
 }
```

Alternatively, I could expand this PR to check if `cond.node` is `Block` and the last `stmts` is `StmtKind::Expr(_)` or `StmtKind::Semi(Expr{node: ExprKind::Ret(Some(_)),..})`. If false, present the proposed suggestion.

After this, if we don't check that the next token is `{`, the output for:

```rust
fn main() {
    if true {
    } else if {
        true
    }
}
```

would be the same as it is now, which is in my eyes reasonable:

```
error: expected `{`, found `}`
 --> file.rs:7:1
  |
7 | }
  | ^
```